react-events/my-app/src/CustomButton.jsx

```diff
@@ -0,0 +1,14 @@
+function CustomButton({ color, text, onCustomClick }) {
+  return (
+    <button
+      onClick={onCustomClick}
```

Comment:
Here is where you would do the callback function and then call `onCustomClick` and pass up `text`